graphql.js

```
@@ -0,0 +1,57 @@
+        .catch((err) => reject(err));
+    });
+  };
+
+  actionHelper.inputToSFAction(node, msg, realAction);
+};
+
+/* Make code available */
+module.exports = function(RED) {
+  function GraphQLQuery(config) {
+
+    const node = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)